src/monodroid/jni/application_dso_stub.cc

```diff
@@ -58,6 +58,9 @@ const ApplicationConfig application_config = {
 	.bundled_assembly_name_width = 0,
 	.number_of_assembly_store_files = 2,
 	.number_of_dso_cache_entries = 2,
+	.android_runtime_jnienv_class_token = 1,
```

Comment:
I wonder if these should be "*really* invalid" values, like 0 or -1.

Reply:
Hm, might be. I forgot that `libxamarin-app.so` produced from this is used in the Android Designer context.